pkg/volume/azure_dd/azure_mounter.go

```diff
@@ -82,9 +83,11 @@ func (m *azureDiskMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return fmt.Errorf("azureDisk - Not a mounting point for disk %s on %s", diskName, dir)
 	}
 
-	if err := os.MkdirAll(dir, 0750); err != nil {
-		glog.Infof("azureDisk - mkdir failed on disk %s on dir: %s (%v)", diskName, dir, err)
-		return err
+	if runtime.GOOS != "windows" {
```

Comment:
Add comment why not check this on windows?
for '/userpath/userpath2' case, do we need to mkdir first?

Reply:
fixed:
// in windows, we will use mklink to mount which will happen in Mount func